management/storageservice/client.go

```diff
@@ -78,6 +78,15 @@ func (s StorageServiceClient) CreateAsync(parameters StorageAccountCreateParamet
 	return s.client.SendAzurePostRequest(azureStorageServiceListURL, data)
 }
 
+func (s StorageServiceClient) DeleteStorageService(serviceName string) (management.OperationID, error) {
```

Comment:
your scale for consistency is inconsistent. other packages do have godoc, so you might as well add it for this one... :P